src/category_theory/epi_mono.lean

```diff
@@ -170,6 +170,27 @@ begin
   apply is_iso.of_mono_retraction,
 end
 
+section
+variables (C)
+
+/-- A split mono category is a category in which every monomorphism is split. -/
+class split_mono_category :=
+(split_mono_of_mono : ∀ {X Y : C} (f : X ⟶ Y) [mono f], split_mono f)
+
+/-- A split epi category is a category in which every epimorphism is split. -/
+class split_epi_category :=
+(split_epi_of_epi : ∀ {X Y : C} (f : X ⟶ Y) [epi f], split_epi f)
+
+end
+
+/-- In a category in which every monomorphism is split, every monomorphism splits. -/
```

Comment:
Do you think it's worth commenting that this shouldn't be an instance? If not that's fine too

Reply:
Done.